std_msgs/Header header

float64[] tau_rls_comp
float64[] tau_pid_comp
float64[] tau_measured
float64[] tau_feedforward
float64[] tau_gravity
float64 mass_rls
float64[] joints_pos